.github/workflows/gh-pages.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
       - name: Get yarn cache
         id: yarn-cache
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Comment:
Thanks for creating the change, maybe we should also quote the $GITHUB_OUTPUT part as per the instruction:
```
run: echo "dir=$(yarn cache dir)" >> "$GITHUB_OUTPUT"
```

Reply:
Good point! Done in [b19b4c5](https://github.com/facebook/memlab/pull/108/commits/b19b4c5d0db13b8346a0b5677aee79f67c948941).